modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -925,14 +963,15 @@ private void scheduleTimers(
      * @param addedNodes Nodes that was added to a topology and should be added to zones data nodes.
      * @param removedNodes Nodes that was removed from a topology and should be removed from zones data nodes.
      * @param revision Revision that triggered that event.
-     * @param saveDataNodes Function that save nodes to a zone's data nodes.
+     * @param saveDataNodesOnScaleUp Function that save nodes to a zone's data nodes.
```

Comment:
Missed `param` for saveDataNodesOnScaleDown

Reply:
fized